fast64_internal/oot/scene/exporter/to_c/scene_table_c.py

```diff
@@ -103,12 +103,17 @@ def getSceneParams(scene, exportInfo, sceneNames):
     # you just have to make it return something other than None, not necessarily a string
     sceneIndex = getSceneIndex(sceneNames, bpy.context.scene.ootSceneExportSettings.option)
     sceneName = sceneTitle = sceneID = sceneUnk10 = sceneUnk12 = None
+    name = scene.name if scene is not None else exportInfo.name
 
     # if the index is None then this is a custom scene
     if sceneIndex is None and scene is not None:
         sceneName = scene.name.lower() + "_scene"
         sceneTitle = "none"
-        sceneID = "SCENE_" + (scene.name.upper() if scene is not None else exportInfo.name.upper())
+        sceneID = ootSceneNameToID.get(name)
+
+        if sceneID is None and bpy.context.scene.ootSceneExportSettings.option == "Custom":
+            sceneID = f"SCENE_{name.upper()}"
```

Comment:
it's weird to use export settings here? like, what if this function is called for something else like deleting a scene or idk

Reply:
I wasn't sure how to check that properly tbh

Reply:
also what happens if `sceneID` stays None